### PR TITLE
docs: Improve Custom Fonts

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/custom-fonts.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
 # Custom Fonts
 
+The `FontFamily` property allows you to customize the font used in you application's UI. Please note that in the following examples `yourfont.ttf` is placeholder for font file name and `Your Font Name` is placeholder for its actual name. To make figuring out the correct format easier, use a font management app. Free application [Character Map UWP](https://www.microsoft.com/store/productId/9WZDNCRDXF41) will provide you with the full string for your selected font:
```
Comment (nickrandolph) [inline doc/articles/features/custom-fonts.md:3]:
```suggestion
The `FontFamily` property allows you to customize the font used in you application's UI. Please note that in the following examples `yourfont.ttf` is placeholder for font file name and `Your Font Name` is placeholder for its actual name. To make figuring out the correct format easier, use a font management app. The free application, [Character Map](https://www.microsoft.com/store/productId/9WZDNCRDXF41), can be used to extract the full string for your selected font:
```

**Review by nickrandolph — COMMENTED** [review]
@MartinZikmund you were mentioning something about on Windows having to include individual font files for each font style (eg italics), whereas for other platforms this can be a single file. Can we extend docs to clarify this?

**Comment by MartinZikmund** [comment]
> @MartinZikmund you were mentioning something about on Windows having to include individual font files for each font style (eg italics), whereas for other platforms this can be a single file. Can we extend docs to clarify this?

That was specific to the default font (e.g. Segoe UI). When you explicitly use a custom font, then having separate font family per weight/style is necessary - so in this case all platforms will behave the same. However, if WinUI were to add support for a "multi-file" font family, we could then improve this as well.

File: doc/articles/features/custom-fonts.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
 # Custom Fonts
 
+The `FontFamily` property allows you to customize the font used in you application's UI. Please note that in the following examples `yourfont.ttf` is placeholder for font file name and `Your Font Name` is placeholder for its actual name. To make figuring out the correct format easier, use a font management app. The free application, [Character Map](https://www.microsoft.com/store/productId/9WZDNCRDXF41), can be used to extract the full string for your selected font:
+
```
Comment (ajpinedam) [inline doc/articles/features/custom-fonts.md:4]:
```suggestion
The `FontFamily` property allows you to customize the font used in your application's UI. Please note that in the following examples, `yourfont.ttf` is a placeholder for the font file name, and `Your Font Name` is a placeholder for its actual name. Use a font management app to make figuring out the correct format easier. The free application, [Character Map](https://www.microsoft.com/store/productId/9WZDNCRDXF41), can be used to extract the full string for your selected font:

```

**Review by ajpinedam — APPROVED** [review]
Looks good, just some small corrections

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**